docs/content/doc/usage/issue-pull-request-templates.en-us.md

```diff
@@ -0,0 +1,44 @@
+---
+date: "2017-01-01T16:00:00+02:00"
+title: "Usage: Issue and Pull Request templates"
+slug: "issue-pull-request-templates"
+weight: 15
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "usage"
+    name: "Issue and Pull Request templates"
+    weight: 15
+    identifier: "issue-pull-request-templates"
+---
+
+# Issue and Pull Request Templates
+
+For some projects there are a standard list of questions that users need to be asked
+for creating an issue, or adding a pull request. Gitea supports adding templates to the
+main branch of the repository so that they can autopopulate the form when users are 
+creating issues, and pull requests. This will cut down on the initial back and forth
+of getting some clarifiying details.
+
+Possible file names for issue templates:
+
+```
+"ISSUE_TEMPLATE.md",
```

Comment:
What about making this a bullet-point list instead (aka deleting `"` and `,`)?

Reply:
thanks @kolaente I've updated the PR to match your suggestion.